src/main/scala/nl.knaw.dans.easy.deposit/servlets/DepositServlet.scala

```diff
@@ -249,8 +254,15 @@ class DepositServlet(app: EasyDepositApiApp)
     }
   }
 
-  private def getRequestBodyAsManagedInputStream = {
-    Try { managed(request.getInputStream) }
+  private def getRequestBodyAsManagedInputStream: Try[ManagedResource[ServletInputStream]] = Try {
+    managed(request.getInputStream)
+  }
+
+  private def canUpdate(uuid: UUID): Try[Unit] = {
+    for {userId <- getUserId
+         deposit <- app.getDepositState(userId, uuid)
```

Comment:
This isn't the `deposit` but the `state`, right?

Reply:
so true -.-